manifests/a/AVASTSoftware/AvastAntivirus/21.4.6266.0/AVASTSoftware.AvastAntivirus.installer.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: AVASTSoftware.AvastAntivirus
+PackageVersion: 21.4.6266.0
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://bits.avcdn.net/productfamily_ANTIVIRUS/insttype_FREE/platform_WIN/installertype_FULL/build_RELEASE/
+  InstallerSha256: 1AD11F63734B137E01B7395D44987AE167DB4583E602A5110099F59FC5D7EC97

```

Comment:
```suggestion
  InstallerSha256: 1ad11f63734b137e01b7395d44987ae167db4583e602a5110099f59fc5d7ec97
```
I'm getting this error:
![image](https://user-images.githubusercontent.com/11600822/120694959-af510280-c4a2-11eb-9822-e91351752da0.png)